rebar.config

```diff
@@ -39,9 +39,14 @@
 {profiles, [{test,
              [{deps, [
                      {meck, "", {git, "https://github.com/eproxus/meck.git", {tag, "0.8.2"}}}
-                     ]}
-             ]
-            },
+                     ]},
+             {erl_opts,
+              [{platform_define, "R14", no_callback_support},
+               {platform_define, "^[0-9]+", namespaced_types},
+               {platform_define, "^R1[4|5]", deprecated_crypto},
```

Comment:
We should double-check but I believe the merge algorithm will handle the platform defines anyway. You'd only need to put in `debug_info`, the rest will be merged in.